coriolis/tests/licensing/test_client.py

```diff
@@ -405,4 +477,4 @@ def test_delete_reservation(self, mock_do_req):
             'delete', '/reservations/%s' % self.reservation_id,
             raw_response=True
         )
-        mock_response.raise_for_status.assert_called_once()
+        mock_raise_response_error.assert_called_once_with(mock_response)
```

Comment:
Please also add a case for `delete_reservation` where the response if not `ok`, and the error code is different from `404`.